lib/test_sites/arcgis_client.rb

```diff
@@ -13,6 +13,13 @@ class ArcGISClient
     # rubocop:disable Style/ClassVars
     @@connection = nil
 
+    def self.all_features
+      all['features'].each_with_object({}) do |arcgis_location, acc|
+        mash = NoWarningMash.new(arcgis_location['attributes'])
+        acc[mash.GlobalID] = mash
+      end
+    end
```

Comment:
Could you get rid of this method and move the logic into `.locations`? I don't want ArcGIS' terminology to bleed into ours cause it's confusing enough with multiple location types, let alone adding another synonym.

Reply:
I really think it's cleaner to have it here. The alternative is to have CACLocation get involved in the details of the response coming back from the ArcGiS API, which has a lot of irrelevant junk. This method returns just the data we care about.

Reply:
I'm confused. I meant the existing method on the same class. Here it is on master...

https://github.com/Spark-Bio/covid-storepoint/blob/master/lib/test_sites/arcgis_client.rb#L41-L43

Reply:
OK I made the change. You probably realize this but it changes the behavior of `locations`, which previously returned an array with both the attributes and geometry for each location. Now we return a hash keyed by ID. I think this is fine since we weren't using geometry, so just FYI.